src/general/OptionsMenu.cs

```diff
@@ -1114,4 +1114,10 @@ private void UpdateSelectedLanguage(Settings settings)
             languageSelection.Selected = languages.IndexOf(settings.SelectedLanguage.Value);
         }
     }
+
+    private void OnLogButtonPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        OS.ShellOpen(ProjectSettings.GlobalizePath(Constants.LOGS_FOLDER_NAME));
```

Comment:
Sorry for the double review, just tested now. The path is incorrect as the constant only specifies the folder name, so you may need to add `user://` before `LOGS_FOLDER_NAME`.